spec/components/npq_separation/admin/adjustments_table_component_spec.rb

```diff
@@ -39,14 +39,14 @@
   describe "adjustment total" do
     context "when show_total is false" do
       it { is_expected.not_to have_css "tbody th", text: t(".total") }
-      it { is_expected.not_to have_css "tbody td", text: "£600" }
+      it { is_expected.not_to have_css "tbody td", text: "£0.00" }
     end
 
     context "when show_total is true" do
       let(:show_total) { true }
 
       it { is_expected.to have_css "tbody th", text: t(".total") }
-      it { is_expected.to have_css "tbody th", text: "£600" }
+      it { is_expected.to have_css "tbody th", text: "£0.00" }
```

Comment:
can we have the total adding up to something more than zero?
just to test the total is actually working, and it's not just doing `nil.to_i`